packages/core/src/controllers/TransactionsController.ts

```diff
@@ -13,6 +15,7 @@ export interface TransactionsControllerState {
   transactions: Transaction[]
   coinbaseTransactions: TransactionByYearMap
   transactionsByYear: TransactionByYearMap
+  lastNetworkInView: CaipNetworkId | undefined
```

Comment:
What exactly is this for?

Reply:
it tracks the connected network during the web component lifecycle, it helps to:

- Refetch the txs if the Activity view is open and the chain is changed from the wallet extension.
- Refetch the txs if the chain is changed when the Activity component is not mounted and then mounted/open.
- Prevent to refetch the txs every time the Activity view is opened.